.graphqlrc.yml

```diff
@@ -1,5 +1,5 @@
 schema: graphql/schema.graphql
-documents: graphql/**/*.graphql
+documents: [graphql/**/*.graphql, src/**/*.ts, src/**/*.tsx]
```

Comment:
lets add it to app-template

Reply:
https://github.com/saleor/saleor-app-template/pull/117